pages.es/linux/zypper.md

```diff
@@ -0,0 +1,28 @@
+# zypper
+
+> Utilidad para la gestión de paquetes en SUSE y openSUSE.
+> Más información: <https://en.opensuse.org/SDB:Zypper_manual>
+
+- Actualiza todos los repositorios:
```

Comment:
```suggestion
- Sincroniza la lista de paquetes y versiones disponibles:
```

Reply:
Running a zypper --help in my Tumbleweed with that command that's the text you can read. I though that this text would be more accurate.

Reply:
I understand but we want a literal translation of the main pages, it's easier to keep them in sync. I think you should split this PR into 2, keep this for edit the main `zypper` page and add the Spanish translation; and another one to add the translation of `i3` and additional pages if you want to add more.